dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEventAnalyticsManager.java

```diff
@@ -580,6 +570,44 @@ else if ( params.isOrganisationUnitMode( OrganisationUnitSelectionMode.CHILDREN
         return sql;
     }
 
+    /**
+     * Creates a SQL statement of descendants org units. When there are multiple
+     * levels the "or" operator will be used as junction.
+     *
+     * @param orgUnitAlias
+     * @param dimensionalItems
+     * @param helper
+     * @return a SQL statement in the format "where/and (ax."uidlevel0" in
```

Comment:
Move the long return doc to the main javadoc text.

Reply:
Done